api/src/pcapi/routes/pro/providers.py

```diff
@@ -29,4 +30,4 @@ def get_providers_by_venue(venue_id: str):
         del provider_dict["authToken"]
         del provider_dict["pricesInCents"]
         result.append(provider_dict)
-    return jsonify(result)
+    return serializers.ListProviderResponse(__root__=result)
```

Comment:
On devrait pouvoir ajouter `orm_mode = True` au modèle de réponse `ProviderResponse`. On n'aurait alors plus besoin d'utiliser as_dict()`. Ça donnerait quelque chose comme ça : 


```suggestion
    return serializers.ListProviderResponse(__root__=[ProviderResponse.from_orm(provider) for provider in providers)
```